packages/host/tests/integration/components/serialization-test.gts

```diff
@@ -5541,7 +5553,6 @@ module('Integration | serialization', function (hooks) {
           attributes: { firstName: 'Burcu', title: null },
           relationships: {
             friend: { links: { self: null } },
-            friendPets: { links: { self: null } },
```

Comment:
here we are providing serialized data for a computed and actually making assertions that the serialized data we provided stomped on top of the computed! which is literally the bug CS-8486